.travis.yml

```diff
@@ -1,30 +1,24 @@
 language: go
 
-dist: trusty
-
-go:
-- "1.8"
-- "1.9"
-- "1.10"
-- tip
+dist: xenial
+sudo: true
+
+matrix:
+  include:
+  - go: "1.10.x"
+    script:
+      - go test -v ./...
+      - goreleaser --snapshot --skip-sign
+  - go: "1.11.x"
+    script:
+      - go test -v ./...
+      - GO111MODULE=on go build -mod=vendor -o=exoip cmd/exoip/main.go
```

Comment:
Why use `-mod=vendor`? We could get caching for free the day we share the cache directory between invocation.

Reply:
OK, I didn't see you committed `vendor/` into VCS.

Reply:
so I guess the other comments regarding the `grep -v "/vendor/"` are obsolete.